web/test/pages/dashboard.test.tsx

```diff
@@ -668,33 +668,173 @@ describe("dashboard page", () => {
       }
     );
 
-    it.each(operatingPhasesWithQuickActions)(
-      "displays state-contracting-external-link quick action when industry id is not home-contractor for %s",
-      (phase) => {
-        useMockProfileData({ operatingPhase: phase, industryId: "some-value" });
-        renderDashboardPage({
-          quickActionLinks: [
-            generateQuickActionLink({
-              name: "some name",
-              filename: "state-contracting-external-link",
-              externalRoute: "some-external-route",
-            }),
-          ],
-        });
-        expect(screen.getByText("some name")).toBeInTheDocument();
-      }
-    );
+    describe("state-contracting-external-link", () => {
```

Comment:
[pebble] I think this can be restructured to be more readable. This allows decreases the length of each describe statement and groups them appropriately.
```
describe("state-contracting-external-link", () => {
  describe("when business persona is starting"){
    ... 
  }
   describe("when business persona is owning"){
    ... 
  }
  describe("when business persona is foreign"){
    ... 
  }
}

Reply:
This might also help pull some of the test setup into a before each or create setup functions to pass in the appropriate values needed for a specific test.

Reply:
I agree